framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -2417,6 +2417,31 @@ protected boolean checkOverride(
         // Get the type of the overriding method.
         AnnotatedExecutableType overrider = atypeFactory.getAnnotatedType(overriderTree);
 
+        // Call the other version of the method, which takes overrider. The split is
```

Comment:
Would some version of this comment and an `@see` be useful in the javadoc of this method?
I'm not sure what you mean with the compatibility reasons, can you expand?
Can you add a brief comment about which of the two methods to override in which situation?
Also the `the other method used to include this call` doesn't make sense to me. The other method didn't exist so far, so couldn't include this call.

Reply:
I'll add something to the javadoc that differentiates the two and suggests when it is appropriate to call one or the other. 

I agree that this comment could be better explained. By "compatibility" I really meant "backwards-compatibility" - I wanted to make sure that calls to the original method would still behave identically, and this comment is trying to explain that. I'll revise it.